proxy/logging/LogBuffer.cc

```diff
@@ -519,6 +519,10 @@ LogBuffer::resolve_custom_entry(LogFieldList *fieldlist, char *printf_str, char
              printf_str == nullptr ? "*NULL*" : printf_str, i, printf_len, markCount,
              alt_printf_str == nullptr ? "*NULL*" : alt_printf_str);
         bytes_written = 0;
+        auto bs       = fieldlist->badSymbols();
+        if (bs.size() > 0) {
+          Note("Excess field markers likely due to bad symbols (%*s) in log format.", static_cast<int>(bs.size()), bs.data());
```

Comment:
Rather than making a separate `Note`, why not attach this directly to the previous `Note`? That would seem less  cryptic. As with the other fields, the 'bad symbols' could always be logged and simply be empty if empty.